src/ResourceManager/Billing/Commands.Billing/Microsoft.Azure.Commands.Billing.dll-Help.psd1

```diff
@@ -0,0 +1,109 @@
+#
```

Comment:
We no longer do help this way.  Please follow the instructions here: https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md  for creating a markdown help file from your current help.xml.  You can remove this file from the PR.